webapp/schemas.py

```diff
@@ -106,6 +105,7 @@ class NoticeImportSchema(NoticeSchema):
 
 
 class NoticeAPISchema(NoticeSchema):
+    type = String()
```

Comment:
In this situation it looks ok, but `type` is an actual [python function](https://docs.python.org/3/library/functions.html#type). So this could cause issues if you manipulate it in the future e.g. pass it as a function argument.